### ISSUE TITLE
Test client downloads with large file sizes

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-client#767]
We want to ensure that the client works no less reliably than downloading via Tor Browser when downloading large files (up to the 500 MB limit). This can be broken down into:

- reliability of the API's `/download` endpoints over Tor
- reliability of the download via `securedrop-proxy`
- reliability of the client's own download/retry behavior

Testing within the client is an easy way to get a sense of the overall scope of any current reliability issues. Testing of the API endpoints will help us narrow down if server-side changes are required for 1.2.1 (e.g., support for range requests, tracked in https://github.com/freedomofpress/securedrop/issues/3907).

**Comment by eloquence** [comment]
For the 2/5-2/19 sprint @kushaldas has offered to do some manual testing & profiling in the client to get us some initial data on reliability.

**Comment by eloquence** [comment]
Kushal reported some results [here](https://github.com/freedomofpress/securedrop-workstation/issues/399#issuecomment-583384437).

**Comment by redshiftzero** [comment]
To figure out if we need to push the server release I did some testing of this today. I tried downloading files for files in the 130 MB, 290 MB, and 450 MB size ranges (1 per file size). It worked well for me (each worked on the first try). This was done on latest versions of all packages as of the date of this comment in a Qubes test environment running on the workstation staging scenario against a staging app server. I investigated the KeyError @kushaldas reported and have an explanation [here](https://github.com/freedomofpress/securedrop-workstation/issues/399#issuecomment-584310420) and it should be resolved if we fix https://github.com/freedomofpress/securedrop-sdk/issues/111. I'm going to close this and we can do more testing of file downloads during the workstation QA period as part of the regular test plan.
